test/mx/distribution/test_mx_distribution_inference.py

```diff
@@ -1291,7 +1291,7 @@ def test_inflated_poisson_likelihood(
 
 
 @pytest.mark.timeout(150)
-@pytest.mark.flaky(max_runs=6, min_passes=1)
+@pytest.mark.flaky(retries=3)
```

Comment:
Could you make sure that the number of trials is unchanged?

Reply:
updated